server/src/main/java/org/eclipse/openvsx/publish/PublishExtensionVersionHandler.java

```diff
@@ -116,8 +129,20 @@ private ExtensionVersion createExtensionVersion(ExtensionProcessor processor, Us
                                 + " is already published"
                                 + (existingVersion.isActive() ? "." : ", but is currently inactive and therefore not visible."));
             }
+            if (mirror == null) {
+                extension.setLastUpdatedDate(extVersion.getTimestamp());
+            } else {
+                // published date should point to timestamp of very first extension version
```

Comment:
Are you mirroring versions out of order?

Reply:
Kind of, order should be ensured between all targets, but since we don't know which target is latest we need to double check.

Reply:
I tested this. Can confirm that versions are mirrored by timestamp. I'm reverting this logic for better performance.
[mirror-rust-analyzer.txt](https://github.com/eclipse/openvsx/files/10392860/mirror-rust-analyzer.txt)